GameModes/StoryGameMode.cs

```diff
@@ -55,8 +55,15 @@ public override ProcessManager.ProcessID MenuProcessId()
             return RainMeadow.Ext_ProcessID.StoryMenu;
         }
 
+        public HashSet<PlacedObject.Type> disallowedPlacedObjects = new()
+        {
+            PlacedObject.Type.SporePlant,  // crashes the game, ask Turtle
+            PlacedObject.Type.HangingPearls,  // duplicates and needs to be synced, ask choc
+        };
+
         public override bool AllowedInMode(PlacedObject item)
         {
+            if (disallowedPlacedObjects.Contains(item.type)) return false;
```

Comment:
I tried this in arena too, didn't seem to work. Just so you're aware. Was gonna investigate more.

Reply:
as in, it still places the object? or it doesn't fix the crash?

Reply:
For arena it still placed the object. I'm going to go test again rq just to verify tho.